File: clearclaim-frontend/src/models/DashboardStats.ts
// DashboardStats.ts
// Represents the admin dashboard statistics
// Returned by GET /api/admin/dashboard

export interface DashboardStats {
  totalCustomers: number;
  activePolicies: number;
  pendingClaims: number;
  approvedClaims: number;
  rejectedClaims: number;
  totalRevenue: number;
  totalHospitals: number;
  totalPlans: number;
}

File: Medical-Insurance/Com.Application.Domain.Entities/DashboardStats.cs
namespace Com.Application.Domain.Entities
{
    public class DashboardStats : BaseEntity
    {
        public int TotalCustomers { get; set; }
        public int ActivePolicies { get; set; }
        public int PendingClaims { get; set; }
        public int ApprovedClaims { get; set; }
        public int RejectedClaims { get; set; }
        public decimal TotalRevenue { get; set; }
        public int TotalHospitals { get; set; }
        public int TotalPlans { get; set; }
    }
}